var/spack/repos/builtin/packages/py-backports-weakref/package.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyBackportsWeakref(PythonPackage):
+    """Backports of new features in Python's weakref module"""
+
+    homepage = "https://github.com/PiDelport/backports.weakref"
+    # Can't use pypi link as it doesn't have source files for the versions
+    # indicated below.
+    url      = "https://github.com/PiDelport/backports.weakref/archive/v1.0.tar.gz"
+
+    version('1.0rc1', sha256='7826e249a0ccb382f1200e5f03c70a624537d4619f050e1c024ff18315550eed')
```

Comment:
I wouldn't add `1.0rc1` since it's a release candidate. 1.0 should be more stable.

Reply:
I added that specific version to satisfy dependency list in a requirements file. Now I think about it is probably not a critical dependency so it should be fine to use version 1.0.post1 I guess.